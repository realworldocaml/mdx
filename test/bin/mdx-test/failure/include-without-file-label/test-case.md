Explicitly setting the block-type to include has little benefits except
for warning you that the `file=...` label is mandatory.

<!-- $MDX type=include -->
```ocaml
```
