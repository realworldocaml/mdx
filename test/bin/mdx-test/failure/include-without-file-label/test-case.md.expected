[mdx] Fatal error: File "test-case.md", lines 4-6: invalid code block: `file` label is required for include blocks.
